src/pages/components/leadspace.mdx

```diff
@@ -29,7 +28,23 @@ import ComponentFeedback from "components/ComponentFeedback";
 
 This is the most commonly used Lead space style. The left-aligned type draws the users' attention to the Lead space messaging immediately. Used in marketing and product pages, among others.
 
-![Lead space left-aligned](../../images/pattern/leadspace/Leadspace_Demo_Centered.jpg)
+There are four available height sizes:
```

Comment:
There are 4 available heights:

Reply:
I think.... ^ did anyone check IBM Styles guidance on this?

Reply:
@shixiedesign 

`I'm confused about when content goes to "Pattern" page and when it goes to "Components" page. Usage info such as height variants should go here, no?`

Usage info such as height variants should go to the components page. The Patterns section contains high-level best practices when using components in combinations of other components, layouts, and flows that address common user objectives. Therefore, any tailored information at the component level will live on the components page.

"React icon on the card broke" 
Not sure how did this happen in preview, but it works fine on the [live website](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/leadspace#feedback).

![Screen Shot 2021-04-13 at 1 50 50 PM](https://user-images.githubusercontent.com/45213839/114605127-5d88ba80-9c5f-11eb-9c28-1167a0292a8e.png)